Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/ReflectionAdapter.h

```diff
@@ -60,6 +62,8 @@ namespace AZ::DocumentPropertyEditor
         //! property editor instances has altered its value.
         void NotifyPropertyChanged(const PropertyChangeInfo& changeInfo);
 
+        virtual void OnBeginRow(AdapterBuilder*, AZStd::string_view);
```

Comment:
I like this, it's pleasantly generic / multi-purpose

Reply:
Hmm, yeah, this gave me pause but I think I'm peaceful with this as a general mechanism in the reflection adapter - `AdapterBuilder` isn't meant to be generally required for building DPEs (it's just a bag of utility functions to make building the tree easier), but it should be fine to expose it as an implementation detail for plugging into `ReflectionAdapter` specifically.